server/routes/overview/workAndInterestsView.ts

```diff
@@ -7,11 +7,16 @@ export default class WorkAndInterestsView {
     tab: string
     prisonerSummary: PrisonerSummary
     workAndInterests: WorkAndInterests
+    hasAtLeastOneSpecificJobRole: boolean
   } {
     return {
       tab: 'work-and-interests',
       prisonerSummary: this.prisonerSummary,
       workAndInterests: this.workAndInterests,
+      hasAtLeastOneSpecificJobRole:
```

Comment:
Good way of checking for this!

Reply:
Even though I've just nuked it as we don't need it anymore! 🤣